src/utils/hooks/useUpdateReviewResponseNEW.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  ReviewResponse,
+  ReviewResponseDecision,
+  ReviewResponseRecommendedApplicantVisibility,
+  useUpdateReviewResponseMutation,
+} from '../generated/graphql'
+
+// below lines are used to get return type of the function that is returned by useUpdateReviewResponseMutation
+type UseUpdateReviewMutationReturnType = ReturnType<typeof useUpdateReviewResponseMutation>
+type promiseReturnType = ReturnType<UseUpdateReviewMutationReturnType[0]>
+
+// hook used to update review response, comment and decision, use as per type definition belo
+// also computes and updates recommendedApplicantVisibility
+type useUpdateReviewResponse = (
```

Comment:
Same here -- capitalise type to `UseUpdateReviewResponse` ?

Reply:
7979039a198be6cc7dfb1299e935770add5a1bd0, how did that even work lol, looks like type and variables can share names